galaxy.yml

```diff
@@ -103,6 +103,7 @@ dependencies:
   "community.kubernetes": "1.2.1"
   "community.libvirt": "*"
   "containers.podman": "*"
+  "kubernetes.core": "3.0.0"
```

Comment:
This should match the RPM package version available (currently [2.3.2](https://softwarefactory-project.io/r/plugins/gitiles/ansible-collection-kubernetes-core)) so both installation methods (RPM and collections) use the same versions

Reply:
Ok, let's use 2.3.2 then. Seems like latest Ansible binary depends on kubernetes.core < 3, and it can't be updated.